tests/lookup/test_decimalfield.py

```diff
@@ -0,0 +1,49 @@
+from django.db.models.aggregates import Sum
+from django.db.models.expressions import F
+from django.test import TestCase
+
+from .models import Product, Stock
+
+
+class DecimalFieldLookupTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.p1 = Product.objects.create(name="Product1", qty_target=10)
+        Stock.objects.create(product=cls.p1, qty_available=5)
+        Stock.objects.create(product=cls.p1, qty_available=6)
+
+        cls.p2 = Product.objects.create(name="Product2", qty_target=10)
+        Stock.objects.create(product=cls.p2, qty_available=5)
+        Stock.objects.create(product=cls.p2, qty_available=5)
+
+        cls.p3 = Product.objects.create(name="Product3", qty_target=10)
+        Stock.objects.create(product=cls.p3, qty_available=5)
+        Stock.objects.create(product=cls.p3, qty_available=4)
+
+    def test_gt(self):
+        qs = Product.objects.annotate(qty_available_sum=Sum('stock__qty_available'))\
```

Comment:
See my previous review for indentation style of this. Perhaps the common qs stuff before the last filter can be moved to setUpTestData.